src/unlimited.cpp

```diff
@@ -61,6 +61,24 @@ std::string ExcessiveBlockValidator(const unsigned int& value,unsigned int* item
   return std::string();
 }
 
+std::string MiningBlockSizeValidator(const uint64_t& value,uint64_t* item,bool validate)
+{
+  if (validate)
+    {
+      if (value > excessiveBlockSize) 
+	{
+        std::ostringstream ret;
+        ret << "Sorry, your excessive block size (" << excessiveBlockSize << ") is smaller than your proposed mined block size (" << value << ").  This would cause you to orphan your own blocks.";    
+        return ret.str();
+	}
+    }
+  else  // Do anything to "take" the new value
```

Comment:
to clearly document that no code is needed there, and to provide a template in case code IS needed...